jobs/JEVS_AQM_STATS

```diff
@@ -62,9 +60,9 @@ export mod_ver=${mod_ver:-${aqm_ver}}
 export VDATE=${VDATE:-${PDYm3}}
 
 export COMIN=${COMIN:-$(compath.py ${envir}/com/${NET}/${evs_ver})}
-export COMINobs=${COMINobs:-${DCOMROOT}}
+export DCOMIN=${DCOMIN:-${DCOMROOT}}
 export COMINaqm=${COMINaqm:-$(compath.py ${envir}/com/${modsys}/${mod_ver})}
-export COMINaqmproc=${COMINaqmproc:-$COMIN/prep/${COMPONENT}}
+export EVSINaqm=${EVSINaqm:-$COMIN/prep/${COMPONENT}}
```

Comment:
Related to @AliciaBentley-NOAA 's comment, we should have EVSINaqm consistent in JEVS_AQM_STATS and JEVS_AQM_PLOTS. Right now they are different. One uses COMIN and one uses compath.py
In stats: `export EVSINaqm=${EVSINaqm:-$COMIN/prep/${COMPONENT}}`
In plots: `export EVSINaqm=${EVSINaqm:-$(compath.py ${envir}/com/${NET}/${evs_ver}/stats/${COMPONENT}/${MODELNAME})}`

Reply:
@AliciaBentley-NOAA @ShelleyMelchior-NOAA even though one is a plot job and one is a stat job, must they refer to the same thing?  

EVSINaqm in stats refers to the prep directory.
EVSINaqm in plots refers to the stats directory.

If they need to be different because they refer to different things, I will make that change.

Reply:
I just mean the start of the path .. one is using COMIN to define the path and one is using compath.py. I don't care how far down into the path the variable goes, though.

Reply:
My question only tangentially relates to your point.  I need to know if variables need to be consistent across prep, stats, and plots?  Or can EVSINaqm refer to a different directory in stats than EVSINaqm refers to in plots?

Reply:
I don't have a problem if they are different b/t $STEP in terms of the definition further down in the path since they have unique drivers, J-jobs, and scripts.

Reply:
OK, I made the requested change.